src/components/chart/state-history-charts.ts

```diff
@@ -152,6 +172,21 @@ class StateHistoryCharts extends LitElement {
     return !(changedProps.size === 1 && changedProps.has("hass"));
   }
 
+  protected updated(changedProps: PropertyValues) {
+    if (changedProps.has("_chartCount")) {
+      if (this._chartCount < this._childYWidths.length) {
+        this._childYWidths.length = this._chartCount;
+        this._maxYWidth =
+          this._childYWidths.length === 0 ? 0 : Math.max(...this._childYWidths);
+      }
+    }
+  }
+
+  private _yWidthChanged(e) {
```

Comment:
```suggestion
  private _yWidthChanged(e: CustomEvent<HASSDomEvents["y-width-changed"]>) {
```

Reply:
Tested this suggestion, works fine, lint clean, prettier clean, etc. Hesitated to touch it since you already approved, not sure if you were in process of merging. You can safely push this change in if you want.